arangod/GeneralServer/AcceptorTcp.cpp

```diff
@@ -131,13 +131,13 @@ void AcceptorTcp<T>::close() {
     _open = false;  // make sure the _open flag is `false` before we
                     // cancel/close the acceptor, since otherwise the
                     // handleError method will restart async_accept.
-    _acceptor.close();
+    _ctx.io_context.wrap([this]() { _acceptor.close(); });
```

Comment:
Same comment as in the equivalent devel PR: should we do the same changes in the AcceptorUnixDomain as well?